lib/SetAPI/assembly/AssemblySetInterfaceV1.py

```diff
@@ -53,6 +53,7 @@ def save_assembly_set(
         :return: dict containing the new set reference and the set info
         :rtype: dict[str, str | list[str | int | dict[str, Any]]]
         """
+        self.set_interface._check_save_set_params(params)
```

Comment:
Added this generic validation from the SetInterfaceV1 class as it makes more sense to check for required params before checking the minutiae of the request params.